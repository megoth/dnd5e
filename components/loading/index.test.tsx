import React from "react";
import { render } from "@testing-library/react";
import Loading from "./index";

describe("Loading", () => {
  it("renders", () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
