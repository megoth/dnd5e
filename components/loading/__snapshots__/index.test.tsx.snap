// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders 1`] = `
<DocumentFragment>
  <div
    data-testid="loading"
  >
    Loading...
  </div>
</DocumentFragment>
`;
